es6-promises/index.js

```diff
@@ -0,0 +1,11 @@
+const takeAChance = require('./take-a-chance');
+
+const promise = takeAChance('Junou');
+
+promise.then((message) => {
+  console.log(message);
+});
+
+promise.catch((error) => {
```

Comment:
This works but you should get in the habit of chaining the calls: `.then().catch()`